modules/infrastructure-moving-monitoring.adoc

```diff
@@ -25,30 +25,59 @@ data:
     alertmanagerMain:
       nodeSelector:
         node-role.kubernetes.io/infra: ""
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/infra
+        operator: Exists
     prometheusK8s:
       nodeSelector:
         node-role.kubernetes.io/infra: ""
-    prometheusOperator:
-      nodeSelector:
```

Comment:
why remove prometheusOperator
we have the setting in: https://docs.openshift.com/container-platform/4.10/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-monitoring_creating-infrastructure-machinesets
and this PR wants to fit the doc issue for prometheusOperator

Reply:
@simonpasquier - could you please comment on @juzhao's question?

Reply:
i agree with @juzhao

Reply:
My understanding is that the change makes sense in general because the current config example to move the monitoring components to infra nodes doesn't work (specifying `tolerations` is required). But we want to keep the `prometheusOperator:` entry as it is a supported option.

Reply:
to be precise, I think that the updated entry for prometheusOperator should be:

```
    prometheusOperator:
      nodeSelector:
        node-role.kubernetes.io/infra: ""
      tolerations:
      - effect: NoSchedule
        key: node-role.kubernetes.io/infra
        operator: Exists
```